setup.bat

```diff
@@ -0,0 +1 @@
+mvn package
```

Comment:
now that I think about it, a setup script for windows is useless. It's only build servers which use our setup.sh command, and they run on Linux.